pkgs/desktops/plasma-5/srcs.nix

```diff
@@ -234,6 +234,14 @@
       name = "plasma-browser-integration-5.14.5.tar.xz";
     };
   };
+  # TODO: Change to use "${mirror}" once available
+  plasma-phone = {
+     version = "5.14.5";
+     src = fetchurl {
+       url = "https://invent.kde.org/kde/plasma-phone-components/-/archive/master/plasma-phone-components-master.tar.gz";
+       sha256 = "0jmw0c2pj4dmak1w49r4lwwg3c4zvgn5d3h3ihrzksh23ni7kb3b";
+     };
```

Comment:
(Nothing to do right now, but to be done eventually before merging.)

Their `invent` platform is a GitLab server, `fetchFromGitLab` should be used to fetch their tarballs since revision tarballs, from most repo hosting systems, might eventually change as they are generated, making the sha256 unstable.

Reply:
What would be the process of getting this on a mirror to match all the other packages in `srcs.nix`?

Reply:
This bit

 * https://github.com/NixOS/nixpkgs/blob/803e956919ef1c14a0403250201d510c36ace64e/maintainers/scripts/fetch-kde-qt.sh#L6-L13

Ends up using either of:

 * https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/kde/fetch.sh
 * https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/plasma-5/fetch.sh

Depending of where exactly it would go (among other `srcs.nix` handled by that script). So basically whenever kde/plasma publishes a released source for that.